packages/iov-bns/src/decode.spec.ts

```diff
@@ -359,5 +362,71 @@ describe("Decode", () => {
       expect(parsed.payload.chainId).toEqual("wonderland");
       expect(parsed.payload.address).toEqual("0xAABB001122DD");
     });
+
+    it("works for CreateMultisignature", () => {
+      // tslint:disable-next-line:readonly-array
```

Comment:
Why not use `readonly codecImpl.multisig.IParticipant[]`?

Reply:
Because `codecImpl.multisig.ICreateContractMsg` expects a non-readonly array: ` Type 'readonly IParticipant[]' is missing the following properties from type 'IParticipant[]': pop, push, reverse, shift, and 6 more.`

Reply:
Interesting, thanks for checking